js/base/Exchange.js

```diff
@@ -1538,6 +1538,20 @@ module.exports = class Exchange {
         return result;
     }
 
+    async fetchL1OrderBooks (symbols, params = {}) {
```

Comment:
Since we're adding `fetchL1Orderbooks` (plural), shouldn't we also add `fetchL1OrderBook` for the sake of consistency? 

One more thing, this is missing from `ccxt.d.ts`, right?

Reply:
I've thought about that too, but as all* exchanges do support singular `fetchOrderBook` there is no need to have fetchL1OrderBook.   

About ccxt.d.ts - maybe it should be added once we have one or two more exchanges with this method implemented (and when `parseL1OrderBook(s)` method is defined finally in base).